app/cdap/components/GridTable/index.tsx

```diff
@@ -209,16 +159,16 @@ export default function GridTable() {
     const rawData: IExecuteAPIResponse = gridData;
     const headersData = createHeadersData(rawData.headers, rawData.types);
     setHeadersNamesList(headersData);
-    if (rawData && rawData.summary && rawData.summary.statistics) {
-      const missingData = createMissingData(gridData?.summary.statistics);
+    if (rawData && rawData?.summary && rawData?.summary?.statistics) {
+      const missingData = createMissingData(gridData?.summary?.statistics);
       setMissingDataList(missingData);
     }
     const rowData =
       rawData &&
```

Comment:
add type for rowData

Reply:
RowData is dynamically coming from API & key, values pairs are completely unexpected prior to the response.